conjure-java-core/src/main/java/com/palantir/conjure/java/types/EndpointErrorGenerator.java

```diff
@@ -55,10 +58,18 @@ public EndpointErrorGenerator(Options options) {
 
     @Override
     public Stream<JavaFile> generate(ConjureDefinition definition) {
+        DeclaredEndpointErrors endpointErrors = DeclaredEndpointErrors.from(definition);
+        if (!options.dangerousDoNotUseEnableEndpointAssociatedErrors()
+                && !endpointErrors.errors().isEmpty()) {
+            List<String> errorNames =
+                    endpointErrors.errors().stream().map(ErrorTypeName::getName).toList();
+            throw new SafeIllegalStateException(
+                    "Errors are associated with endpoints. This feature is currently not supported.",
```

Comment:
Proposal on a slightly more explicit message, to help with potential OSS consumers that would have already enabled this
```suggestion
                    "Errors are associated with endpoints. This feature has been temporarily disabled while we iterate through issues with its rollout. If this is blocking you or you need this immediately, please reach out to the Conjure maintainers at https://github.com/palantir/conjure-java/issues so we can help support your needs.",
```